crypto/bio/bss_file.c

```diff
@@ -89,6 +89,10 @@
 
 # if !defined(OPENSSL_NO_STDIO)
 
+#  if defined(OPENSSL_SYS_MSDOS) && !defined(OPENSSL_SYS_WINDOWS)
+#   include <libc/unconst.h>
+static void dosify_filename(const char *filename);
+#  endif
```

Comment:
> considering file_fopen is already an #if sallad, I'm not sure if suddenly using a radically different solution for DJGPP makes sense right now

I'd say that creating meaningful precedent is not inappropriate at *any* time. And I don't really see that it would be bigger job, but let it be your call...